### PR TITLE
[Term Entry] Kotlin Math .sqrt()

### LINKED ISSUE #3631: [Term Entry] Kotlin Math .sqrt()
### Reason/inspiration (optional)

We would like a new term entry on the `.sqrt()` method for Kotlin: [Math](https://www.codecademy.com/resources/docs/kotlin/math-methods). The entry should go in a new file under `docs/content/kotlin/concepts/math-methods/terms/sqrt/sqrt.md`.

The entry should describe what the method does. Be sure to include the following sections:

- A "Syntax" section with a ` ```pseudo` code snippet.
- An "Example" section with a ` ```kotlin `code snippet.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by rahul0x00**
### Description

Closes #3631 
Added new Term Entry for Kotlin Math.sqrt()

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/kotlin/concepts/math-methods/terms/sqrt/sqrt.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+Title: 'Math.sqrt()'
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/sqrt/sqrt.md:2]:
```suggestion
Title: 'sqrt()'
```

File: content/kotlin/concepts/math-methods/terms/sqrt/sqrt.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+Description: 'Calculates the square root of a given number.'
+Subjects:
+  - 'Mathematics'
+  - 'Programming'
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/sqrt/sqrt.md:6]:
```suggestion
  - 'Code Foundations'
```

File: content/kotlin/concepts/math-methods/terms/sqrt/sqrt.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  - 'Computer Science'
+Tags:
+  - 'Math'
+  - 'Functions'
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+The **`Math.sqrt()`** method in Kotlin's `Math` class is used to calculate the square root of a given number.
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/sqrt/sqrt.md:16]:
```suggestion
The **`math.sqrt()`** method in Kotlin's `math` class is used to calculate the square root of a given number.
```

File: content/kotlin/concepts/math-methods/terms/sqrt/sqrt.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+## Syntax
+
+```pseudo
+val squareRoot = Math.sqrt(number)
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/sqrt/sqrt.md:21]:
```suggestion
import kotlin.math.sqrt
val squareRoot = sqrt(number)
```

File: content/kotlin/concepts/math-methods/terms/sqrt/sqrt.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+```
+
+- `number`: The number for which you want to calculate the square root. It should be a non-negative value (zero or a positive number).
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/sqrt/sqrt.md:24]:
```suggestion
- `number`: The number for which you want to calculate the square root. It should be a non-negative value (zero or a positive number) of type `Double` or `Float`.
```

File: content/kotlin/concepts/math-methods/terms/sqrt/sqrt.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+The `Math.sqrt()` method returns a value of type `Double` representing the square root of the input number. The result is a non-negative value or NaN if the input is negative.
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/sqrt/sqrt.md:26]:
```suggestion
The method returns a number, of the same type given, representing the square root. The result is a non-negative value or `NaN` if the input is negative.
```

File: content/kotlin/concepts/math-methods/terms/sqrt/sqrt.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+
+## Example
+
+This example shows how to use the `Math.sqrt()` method to calculate the square root of a number:
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/sqrt/sqrt.md:30]:
```suggestion
This example shows how to use the `sqrt()` method to calculate the square root of a number:
```

File: content/kotlin/concepts/math-methods/terms/sqrt/sqrt.md [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+
+```kotlin
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/sqrt/sqrt.md:32]:
```suggestion
```kotlin
import kotlin.math.sqrt
```

File: content/kotlin/concepts/math-methods/terms/sqrt/sqrt.md [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+fun main() {
+    val number = 25.0
+
+    val squareRoot = Math.sqrt(number)
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/sqrt/sqrt.md:36]:
```suggestion
    val squareRoot = sqrt(number)
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@rahul0x00 I've completed my initial review. Please update the entry, and tag me when you've finished making changes. Thanks

**Comment by rahul0x00** [comment]
@caupolicandiaz updated the entry

**Comment by aman79lb** [comment]
brother , how to pass checks and why my pull request is not getting reviewed , it is named " Create lambda.md " in the pull requests.

**Comment by rahul0x00** [comment]
> brother , how to pass checks and why my pull request is not getting reviewed , it is named " Create lambda.md " in the pull requests.

Because you've put your PR in draft

**Review by caupolicandiaz — APPROVED** [review]
@rahul0x00 This is ready for a second review.

**Review by SSwiniarski — APPROVED** [review]
@rahul0x00 Made one change to remove an instance of "you" and this looks good to go.

**Pull request merged**